### PR TITLE
add commit

### PR DESCRIPTION
**Pull request by Ekamba**

### CODE REVIEW

File: react/week2/src/AddTodo.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from "react";
+
+import TimeSpend from "./TimeSpend";
+
+function Form(props) {
```
Comment (Daryan-Tech) [inline react/week2/src/AddTodo.js]: just a tip, under this line you can do this:
const { onSubmit, value } = props;  this is easier to use the props and also if accidentally you create a variable with the same name that will exist in the props, it prevents you to do that.

**Comment by Daryan-Tech** [comment]
good job, there is some comment on the code please check.
also for the design:
please don't include the ids inside the application design, it is not so useful to have and will take up lot of space :)

File: react/week2/src/AddTodo.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <div>
+      <div class="bg-info clearfix">
+        <button type="button" class="btn btn-secondary float-left">
+          <h3 align="left">
+            {" "}
```
[the remark this answers is not in the document]
Comment (Ekamba) [reply react/week2/src/AddTodo.js]: Hi Daryan, thank for reviewing my codes. It is bootstrap style.

**Comment by Ekamba** [comment]
I did the No items on the title instead, these snippets below handle that: 
`useEffect(() => {
    let count = 0;
    todos.map(todo => (!todo.done ? count++ : null));
    document.title = `${count} task${count > 1 ? "s" : ""} todo`;
  });`

**Comment by Daryan-Tech** [comment]
> I did the No items on the title instead, these snippets below handle that:
> `useEffect(() => { let count = 0; todos.map(todo => (!todo.done ? count++ : null)); document.title = `${count} task${count > 1 ? "s" : ""} todo`; });`

oh okay, but please do not ever do this, it is a much better way to render a span tag maybe with the text  "there is no active Todo item"

File: react/week2/src/AddTodo.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Daryan-Tech) [reply react/week2/src/AddTodo.js]: > Hi Daryan, thank for reviewing my codes. It is bootstrap style.

okay, but it doesn't meant that you shouldn't remove it if it is useless in this case it is useless if you remove this nothing will happen so please do it.
You are so welcome :)

File: react/week2/src/AddTodo.js (hunk 1 above)
Comment (Ekamba) [reply react/week2/src/AddTodo.js]: thanks for advice.

> 
> 
> > I did the No items on the title instead, these snippets below handle that:
> > `useEffect(() => { let count = 0; todos.map(todo => (!todo.done ? count++ : null)); document.title = `${count} task${count > 1 ? "s" : ""} todo`; });`
> 
> oh okay, but please try this:
> when there is no todo item, add a span tag below the addTodo button with this text "No items"

Ok, I'll try it wright away. But I have struggle with it and try my best. If you have a suggestion you're welcome :-)

**Pull request merged**